install/macos.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: macOS
+---
+
+# Installing RGBDS on macOS
+
+## Using the Homebrew package manager
+
+:::info Availability
+
+This is not advisable for users on older macOS versions, as binaries ("bottles") are only available for [recent macOS major releases](https://formulae.brew.sh/formula/rgbds).
+Older macOS systems have been reported to compile for several days, but only when using Homebrew.
```

Comment:
This is, again, redundant. We are already in the "Homebrew package manager" subchapter..

Reply:
It is redundant, but was added to clarify that it does not apply to building from source yourself. (Someone was confused by that.)